modules/nw-understanding-networking-securing-network-traffic.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * networking/understanding-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-understanding-networking-securing-network-traffic_{context}"]
+= Securing network traffic
+
+{product-title} provides tools to secure your network by creating rules that control which components are allowed to communicate. This is primarily managed through two types of policy resources: network policies and administrative network policies.
+
+== Network policies
+
+A network policy is a resource that allows you to control the flow of traffic at the IP address or port level. These policies operate at the namespace (project) level. This means they are typically managed by developers or project administrators to secure their specific applications.
+
+By default, all pods in a project can communicate with each other freely. However, when you apply a `NetworkPolicy` to a pod, it adopts a "default-deny" stance. This means it will reject any connection that is not explicitly allowed by a policy rule. You use labels and selectors to define which pods a policy applies to and what ingress and egress traffic is permitted.
+
+== Administrative network policies
```

Comment:
All titles and section headings must have an anchor ID.

OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings